cms/djangoapps/contentstore/views/course.py

```diff
@@ -555,6 +560,13 @@ def course_listing(request):
     enabled_organizations = get_enabled_organizations(request)
     org = enabled_organizations[0].get('short_name', '') if enabled_organizations else None
 
+    global_course_creator_enabled = WaffleSwitchNamespace(
```

Comment:
I have added a waffle switch `enable_global_course_creator` that can be used to control whether we want to use a global course creator or not. kindly share your thoughts if it should be there or not. @zubair-arbi @ziafazal

Reply:
No. It will only change the course list. I am going to remove this